lib/monitoring/addon/components/enable-monitoring/component.js

```diff
@@ -94,6 +94,9 @@ export default Component.extend(ReservationCheck, {
       answers['prometheus.resources.core.limits.cpu'] = `${ get(this, 'limitsCpu') }m`;
       answers['prometheus.resources.core.requests.memory'] = `${ get(this, 'requestsMemory') }Mi`;
       answers['prometheus.resources.core.limits.memory'] = `${ get(this, 'limitsMemory') }Mi`;
+      if ( !get(this, 'enabled') || get(this, 'useReleaseName') ) {
+        answers['prometheus.persistent.useReleaseName'] = 'true';
```

Comment:
Is there a reason we're using string over bool here?

Reply:
Backend only supports string for now.